test/ClangImporter/availability_open_enums_macos_swift3.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck -verify -I %S/Inputs/custom-modules -swift-version 3 -target x86_64-apple-macosx10.13 %s 
```

Comment:
Any way to test all the Apple platforms? It seems like `@available` could avoid having to hardcode a target triple.